python/vineyard/contrib/ml/tensorflow.py

```diff
@@ -145,6 +156,40 @@ def tf_table_resolver(obj, resolver):
     return tf.data.Dataset.from_tensor_slices((dict(arrow), labels))
 
 
+def tf_global_tensor_resolver(obj, **kw):
+    meta = obj.meta
+    endpoint = kw.get('endpoint')
+    client = vineyard.connect(endpoint)
+    num = from_json(meta['num'])
+    partition_index = from_json(meta['partition_index_'])
+    data = []
+    for i in range(partition_index):
+        loc = meta[f'partition_{i}']
+        if loc.islocal:
+            data.append(client.get(loc.id))
```

Comment:
Resolving a partition in a global object is similar to resolving a tensor in a dataframe. Please refer to (https://github.com/v6d-io/v6d/blob/44bf8f47fca10f2e7559f0865d7285a7a2c49ff4/python/vineyard/data/dataframe.py#L58)
to avoid establishing another client and request to the server which is a waste of resources.